app/views/service_providers/_service_provider_table_header.erb

```diff
@@ -3,7 +3,9 @@
     <th scope="col">Friendly name</th>
     <th scope="col">Issuer</th>
     <th scope="col">IAL</th>
-    <th scope="col">Active</th>
+    <% if !is_prod %>
+      <th scope="col"><abbr title="Accessible in the Integration environment">Accessible</abbr></th>
+    <% end %>
     <th scope="col">Cert Exp</th>
     <% if show_created_at %>
```

Comment:
with my suggestion in `app/views/service_providers/_service_provider_table.erb`, this would become 

```
<% if is_prod %>
```

Reply:
For some reason, that's updating the SP Table rows, but not the header.
I don't fully understand why it works at all, if we're statically setting `is_prod` to `false`?

Reply:
ah, nevermind: I missed an instance of `show_created_at`. I get how it fits together now.

Reply:
oh you know what, i definitely got confused -- the  `created_at` column is flipped on and off depending on whether it's an admin view or a user view (the admin view, can be found here: [admin all service providers view](https://github.com/18F/identity-dashboard/blob/main/app/views/service_providers/all.html.erb#L3). so this won't work, apologies.